README.md

```diff
@@ -82,6 +82,7 @@ Rule                              | Recommended      | Options
 [no-unsafe-spy][]                 | 1                |
 [valid-expect][]                  | 1                |
 [prefer-jasmine-matcher][]        | 1                |
+[prefer-toHaveBeenCalledWith][]   | 0                |
```

Comment:
Hmmm, how about usages like `not.toHaveBeenCalled()` would you also use `not.toHaveBeenCalledWith(...)` instead?